src/scanners/machine_uuid_scanner.go

```diff
@@ -11,17 +11,34 @@ import (
 )
 
 const (
-	DefaultUUID = "00000000-0000-0000-0000-000000000000"
+	DefaultUUID         = "00000000-0000-0000-0000-000000000000"
+	VmwareDefualtSerial = "None"
 )
 
+//go:generate mockery -name Dependencies -inpkg
+type Dependencies interface {
```

Comment:
maybe rename to HWDiscovery ?

Reply:
Don't know.  We do HW in the inventory.  Maybe IdDiscovery or SerialDiscovery?

Reply:
sure